lib/restforce/collection.rb

```diff
@@ -33,6 +33,12 @@ def size
     end
     alias length size
 
+    def count(*args)
+      return size unless block_given? || !args.empty?
```

Comment:
What do you think of adding a comment to explain the "why" behind this implementation - why we need this condition and why we bother to delegate to `#size`? I suspect it could cause some future person to wonder what's going on.